AdyenActions/Components/3DS2/Action handlers/3DS2 Without Delegated Authentication/ThreeDS2CoreActionHandler.swift

```diff
@@ -8,6 +8,20 @@
 import Adyen3DS2
 import Foundation
 
+// TODO: Remove this once 3ds2 sdk is released.
```

Comment:
Should we wait for 3DS release?

Reply:
Yes we should wait before the merge, the opaque error object is available only in the next release.